src/main/java/net/barrage/school/java/ecatalog/app/merchantService/MerchantServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package net.barrage.school.java.ecatalog.app.merchantService;
+
+import lombok.SneakyThrows;
+import net.barrage.school.java.ecatalog.model.Merchant;
+import net.barrage.school.java.ecatalog.model.Product;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+public class MerchantServiceImpl implements MerchantService {
+    @Autowired
+    MerchantRepostiory merchantRepostiory;
```

Comment:
Pls use `private final` and avoid `Autowired` in main code when its possible.

Reply:
`Auotwired` will bring problems to u when u will cover ur code with unit tests. Better to inject beans through constructor.